include/psen_scan_v2/ros_scanner_node.h

```diff
@@ -149,6 +158,29 @@ void ROSScannerNodeT<S>::laserScanCallback(const LaserScan& scan)
   // LCOV_EXCL_STOP
 }
 
+template <typename S>
+void ROSScannerNodeT<S>::publishChangedIOStates(const std::vector<psen_scan_v2_standalone::IOState>& io_states)
+{
+  for (const auto& io : io_states)
+  {
+    if (last_io_state_ != io)
```

Comment:
Is this case ever expected?

Reply:
yes everytime, the io of the device changes? this case is the whole purpose of this PR